src/transformers/models/led/modeling_tf_led.py

```diff
@@ -2028,8 +2062,91 @@ def call(
             encoder_global_attentions=inputs["encoder_outputs"].global_attentions,
         )
 
+
+@add_start_docstrings(
+    "The bare LED Model outputting raw hidden-states without any specific head on top.",
+    LED_START_DOCSTRING,
+)
+class TFLEDModel(TFLEDPreTrainedModel):
```

Comment:
Also `TFLEDModel` never had a `TFLEDPreTrainedModel` as a layer inside -> so why do we have to change `TFLEDModel` in the first place? I though we only need to change `TFLEDForConditionalGeneration`

Reply:
I don't get what you mean by `Also TFLEDModel never had a TFLEDPreTrainedModel as a layer inside` ?

Reply:
AH ok! I just got what you meant! indeed `TFLEDModel` had no `led` attribute that was a model. Do you prefer to keep `TFLEDModel` as it was and use the brand new `TFLEDMainLayer` only in `TFLEDForConditionalGeneration`?

Reply:
If the solution is fully backwards compatible including things, like 

```TFLedModel.from_pretrained("allenai/led-base-16384", from_pt=True)```

it's totally fine for me!